src/material/slide-toggle/_m2-slide-toggle.scss

```diff
@@ -95,10 +95,10 @@
     ),
     density: (
       slide-toggle-state-layer-size: map.get((
-        0: 40px,
-        -1: 36px,
-        -2: 32px,
-        -3: 28px,
+        0: 48px,
```

Comment:
I don't think that this is the right way to solve it, it just makes the ripple larger. The way we handle the touch target in other components is to have a transparent `div` that is always at least 48x48 and isn't affected by density. See `.mat-mdc-radio-touch-target` as an example.

Reply:
That does actually look like a simpler and better solution that either of the others. Started at https://github.com/angular/components/pull/31486.